packages/lib-classifier/src/components/Classifier/components/MetaTools/MetaTools.spec.js

```diff
@@ -1,130 +1,224 @@
-import { FavouritesButton } from '@zooniverse/react-components'
 import React from 'react'
-import { shallow } from 'enzyme'
-import sinon from 'sinon'
-import { MetaTools } from './MetaTools'
-import Metadata from './components/Metadata'
-import CollectionsButton from './components/CollectionsButton'
-import HidePreviousMarksButton from './components/HidePreviousMarksButton'
-import { Factory } from 'rosie'
-import SHOWN_MARKS from '@helpers/shownMarks'
-
-const subjectWithMetadata = Factory.build('subject', { metadata: { foo: 'bar' } })
-
-const favoriteSubject = Factory.build('subject', { favorite: true })
-
-const spy = sinon.spy()
-
-const interactionTask = {
-  hidePreviousMarks: false,
-  marks: [{ x: 0, y: 0 }],
-  shownMarks: SHOWN_MARKS.ALL,
-  togglePreviousMarks: spy
-}
-
-describe('Component > MetaTools', function () {
-  it('should render without crashing', function () {
-    const wrapper = shallow(<MetaTools />)
-    expect(wrapper).to.be.ok()
-  })
+import { render, screen } from '@testing-library/react'
+import asyncStates from '@zooniverse/async-states'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import { when } from 'mobx'
+import { Provider } from 'mobx-react'
+
+import { DrawingTaskFactory, UPPFactory, WorkflowFactory } from '@test/factories'
+import mockStore from '@test/mockStore'
+import MetaTools from './'
+
+describe('Components > MetaTools', function () {
+  function withStore(store) {
+    return function Wrapper({ children }) {
+      return (
+        <Grommet theme={zooTheme}>
+          <Provider classifierStore={store}>
+            {children}
+          </Provider>
+        </Grommet>
+      )
+    }
+  }
+
+  this.timeout(0)
+
+  describe('defaults', function () {
+    let addToCollections, favourite, hidePreviousMarks, metadata
+
+    before(function () {
+      const store = mockStore()
+      render(
+        <MetaTools />,
+        {
+          wrapper: withStore(store)
+        }
+      )
+      addToCollections = screen.getByRole('button', {
+        name: 'MetaTools.CollectionsButton.add'
+      })
+      metadata = screen.getByRole('button', {
+        name: 'MetaTools.MetadataButton.label'
+      })
+      favourite = screen.getByRole('checkbox', {
+        name: 'Add to favorites'
```

Comment:
Is this checkbox name a sentence string rather than a translations key because it's from lib-react-components?